docs/generate_genome_attributes.md

```diff
@@ -0,0 +1,24 @@
+# Generate Genome Attributes
```

Comment:
Can we make this a general document for generating collections data, and so also include taxa counts information (maybe in the future, when the script has been reworked to be general rather than gtdb specific)

Reply:
👍

Reply:
:+1: